src/nile/cli.py

```diff
@@ -84,15 +84,47 @@ def run(path, network):
 
 
 @cli.command()
-@click.argument("artifact", nargs=1)
-@click.argument("arguments", nargs=-1)
-@network_option
+@click.argument("contract_name", nargs=1)
+@click.argument("params", nargs=-1)
+@click.option("--max_fee", nargs=1)
+@click.option("--salt", nargs=1, default=0)
+@click.option("--unique", is_flag=True)
+@click.option("--account")
 @click.option("--alias")
 @click.option("--abi")
+@click.option("--udc")
 @mainnet_token_option
-def deploy(artifact, arguments, network, alias, abi=None, token=None):
+@network_option
+def deploy(
+    contract_name,
+    salt,
+    params,
+    max_fee,
```

Comment:
Shouldn't `max_fee` default to `None` here? I don't think it's passed to `account.deploy_contract`

Reply:
Correct me if I'm wrong, I think that Click behavior is passing the value as None without us needing to set the default in the method (adding unnecessary verbosity IMO). I tested it and this seems to be the behavior (couldn't find the entry in the docs though).

Reply:
It seems you are correct! So yeah, let's remove the unnecessary `None`s